components-api/src/main/scala/pl/touk/nussknacker/engine/api/json/decoders/TypingResultDecoder.scala

```diff
@@ -1,96 +1,14 @@
-package pl.touk.nussknacker.engine.api.typed
+package pl.touk.nussknacker.engine.api.json.decoders
 
 import cats.data.NonEmptyList
 import io.circe._
-import io.circe.Json._
-import pl.touk.nussknacker.engine.api.typed.TypeEncoders.typeField
-import pl.touk.nussknacker.engine.api.typed.TypingType.{decoder, TypingType}
+import pl.touk.nussknacker.engine.api.json.TypingType
+import pl.touk.nussknacker.engine.api.json.TypingType.{TypingType, typeField}
 import pl.touk.nussknacker.engine.api.typed.typing._
-import pl.touk.nussknacker.engine.util.Implicits.RichScalaMap
 
 import scala.collection.immutable.ListMap
 import scala.util.{Failure, Success, Try}
 
-//TODO: refactor way of encoding to easier handle decoding.
-
-// If changes are made to Encoders/Decoders should also change Schemas in NodesApiEndpoints.TypingDtoSchemas for OpenApi
-object TypeEncoders {
```

Comment:
Refactored structure in package `pl.touk.nussknacker.engine.api` in components-api:
- package json
- with sub-packages encoder/decoder

This object moved into encoders package in a separate file